exercises/simple-linked-list/example.js

```diff
@@ -0,0 +1,72 @@
+function ListElement (value, next) {
+  if (!(this instanceof ListElement))
+    throw new Error('ListElement is a constructor.');
+
+  if (value === undefined)
+    throw new Error('Value required.');
+
+  if (next !== undefined && !(next instanceof ListElement))
+    throw new Error('A ListElement instance as next value is required.');
+
+  this.value = value;
+  this.next = next;
+}
+
+function LinkedList () {
+}
+
+LinkedList.prototype.push = function (value) {
+  if (value === undefined) throw new Error('Argument required.');
+
+  var newEl = (value instanceof ListElement)
+    ? value
+    : new ListElement(value)
```

Comment:
Should I use semicolons to end statements. If that's the case, I'll add them across the file. I think I've coded it mostly following the JS Standard Style (no problem on changing this, just want to make sure).